src/org/traccar/model/Command.java

```diff
@@ -47,6 +47,8 @@
     public static final String TYPE_FIRMWARE_UPDATE = "firmwareUpdate";
     public static final String TYPE_SET_CONNECTION = "setConnection";
     public static final String TYPE_SET_ODOMETER = "setOdometer";
+    public static final String TYPE_GET_GSMSTATUS = "getGsmstatus";
```

Comment:
Both named don't follow naming convention. I think it should be `TYPE_GET_MODEM_STATUS` and `TYPE_GET_DEVICE_STATUS`.